README.md

```diff
@@ -1,45 +1,37 @@
 # Satellite-clone
+
 Satellite-clone contains simple Ansible playbooks that can be used to perform any of the following tasks:
-* Setup Satellite 6.1 or 6.2 install with the Satellite backup data.
+* Setup a Satellite 6.1 or 6.2 install with restored backup data.
 * Change the hostname on an existing Satellite install.
-* Perform a minor upgrade of an existing Satellite.
+* Perform a minor upgrade of an existing Satellite. 
 
 ## Getting Started
-Ideally, you need two hosts to run this project:
-
-1. Ansible Control node (referred to as `Control node` in the rest of this document) is the host from which this ansible project is run.
-2. Destination node - This must be one of the following:
-    - A blank machine - to clone an existing Satellite Server.
-    - A Satellite server - for a hostname change or for Updating Satellite.
 
-**Note**
+#### You will need: ####
 
-1. You can get away with using one host by optionally choosing to use the `Destination node` as the `Control node`.
-2. Make sure that the `Control node` can connect to the `Destination node` via paswordless ssh.
+**to clone a Satellite Server**:
+    - A blank (vanilla install) RHEL 7 server. You will run the setup commands here.
```

Comment:
@sthirugn From our scrum conversation, I thought we decided to just support RHEL 7 as the blank target machine?